LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 sbilly
+Copyright (c) 2025 sbilly
```

Comment:
Is it needed if the file wasn't changed?

Reply:
""The copyright year should be updated when the content of the software or documentation has been modified in the new year. This reflects the year in which the changes were made and ensures that the copyright notice is current and accurate.""